server/server.js

```diff
@@ -51,7 +51,7 @@ const startApiGWGauge = () => {
     setInterval(async () => {
         try {
             const res = await fetch(`${API_GATEWAY}/arbeidsgiver-arbeidsforhold-api/internal/actuator/health`, {
-                ...(APIGW_HEADER ? {headers: {'x-nav-apiKey': APIGW_HEADER}} : {})
+                ...(ARBEIDSFORHOLD_API_GW_HEADER ? {headers: {'x-nav-apiKey': APIGW_HEADER}} : {})
```

Comment:
Det virker litt kontraintuitivt for meg at ikke begge forekomstene av `APIGW_HEADER` på denne linjen ikke byttes. Dette skal vel teste det som skjer på linje 98 i denne filen.

Reply:
Jeg har pushet en fiks for dette, samt fjernet den ubrukte sonekrysningsproxyen